lib/utils/index.js

```diff
@@ -537,6 +538,30 @@ class Utils {
   static shouldReplaceStack(err) {
     return !Utils.alwaysDisplayError(err);
   }
+
+
+  /**
+   * Read the User file from the stackTrace and forms 
+   * string with highlighting the line with error.
+   */ 
+  static beautifyStackTrace(err) {
+    let lines= '';
+    if (err && ['NightwatchAssertError'].includes(err.name)) {
```

Comment:
shouldn't we process all types of errors?

Reply:
Like main concern would be, other types of errors could be outside of test files and stack trace would not point to the exact location.

Reply:
I don't think that's a valid concern, I think syntax, type, and reference errors all need to be handled because it will be very useful to see directly where the error is coming from. Also there are several other places where errors are being printed, so maybe we should do the stack trace parsing in Logger.error()?